packages/web-components/fast-foundation/src/anchored-region/anchored-region.ts

```diff
@@ -289,7 +290,7 @@ export class AnchoredRegion extends FASTElement {
     public fixedPlacement: boolean = false;
     private fixedPlacementChanged(): void {
         if (
-            (this as FASTElement).$fastController.isConnected &&
+            (this as FoundationElement).$fastController.isConnected &&
```

Comment:
You could keep this as FASTElement since that's there the controller is defined.

Reply:
Would we want one to be tied to FASTElement even though all others extend Foundation? How do we document when that might be expected? Or, do we categorize things differently in that case?

Reply:
FASTElement is the lowest level base class that provide the `$fastController` functionality. So, if accessing that, I'd probably make the cast to `FASTElement`. It's usually best to cast to the least specific thing that has the features you want. It's not a huge deal here though.